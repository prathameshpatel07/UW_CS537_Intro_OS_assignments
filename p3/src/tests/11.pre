echo "ls tests/p2a-test > /tmp/output11" > tests/11.out 2> tests/11.err
ls tests/p2a-test > /tmp/output11 2> /tmp/output11
echo "cat /tmp/output11" >> tests/11.out 2>> tests/11.err
cat /tmp/output11 >> tests/11.out 2>> tests/11.err
echo "rm -f /tmp/output11" >> tests/11.out 2>> tests/11.err
rm -f /tmp/output11 >> tests/11.out 2>> tests/11.err
echo "exit" >> tests/11.out 2>> tests/11.err

