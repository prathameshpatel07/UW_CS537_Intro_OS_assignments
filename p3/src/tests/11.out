ls tests/p2a-test > /tmp/output11
cat /tmp/output11
test1
test2
test3
test4
rm -f /tmp/output11
exit
